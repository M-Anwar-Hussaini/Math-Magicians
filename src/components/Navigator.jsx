function Navigator() {
  return (
    <div className="container py-4 bg-success mb-2 rounded">
      <nav className="d-flex justify-content-between align-items-center">
        <a href="/" className="btn btn-light">
          <h1>Math Magicians</h1>
        </a>
        <div className="list-group list-group-horizontal">
          <a href="/" className="list-group-item list-group-item-action">
            Home
          </a>
          <a
            href="/calculator"
            className="list-group-item list-group-item-action"
          >
            Calculator
          </a>
          <a href="/quotes" className="list-group-item list-group-item-action">
            Quotes
          </a>
        </div>
      </nav>
    </div>
  );
}

export default Navigator;
