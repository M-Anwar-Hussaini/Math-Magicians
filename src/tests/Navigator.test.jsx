import '@testing-library/jest-dom/extend-expect';
import { screen, render } from '@testing-library/react';
import Navigator from '../components/Navigator';

describe('Calculate', () => {
  it('Renders Navigator component', () => {
    const nav = render(<Navigator />);

    expect(nav.container).toMatchSnapshot();
  });

  it('Renders Navigator component', () => {
    render(<Navigator />);
    const element = screen.getByText('Home');

    expect(element).toBeInTheDocument();
  });
});
