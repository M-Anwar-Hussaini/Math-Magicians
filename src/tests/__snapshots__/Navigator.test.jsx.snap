// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculate Renders Navigator component 1`] = `
<div>
  <div
    class="container py-4 bg-success mb-2 rounded"
  >
    <nav
      class="d-flex justify-content-between align-items-center"
    >
      <a
        class="btn btn-light"
        href="/"
      >
        <h1>
          Math Magicians
        </h1>
      </a>
      <div
        class="list-group list-group-horizontal"
      >
        <a
          class="list-group-item list-group-item-action"
          href="/"
        >
          Home
        </a>
        <a
          class="list-group-item list-group-item-action"
          href="/calculator"
        >
          Calculator
        </a>
        <a
          class="list-group-item list-group-item-action"
          href="/quotes"
        >
          Quotes
        </a>
      </div>
    </nav>
  </div>
</div>
`;

exports[`Calculate Renders Navigator component 2`] = `
<div>
  <div
    class="container py-4 bg-success mb-2 rounded"
  >
    <nav
      class="d-flex justify-content-between align-items-center"
    >
      <a
        class="btn btn-light"
        href="/"
      >
        <h1>
          Math Magicians
        </h1>
      </a>
      <div
        class="list-group list-group-horizontal"
      >
        <a
          class="list-group-item list-group-item-action"
          href="/"
        >
          Home
        </a>
        <a
          class="list-group-item list-group-item-action"
          href="/calculator"
        >
          Calculator
        </a>
        <a
          class="list-group-item list-group-item-action"
          href="/quotes"
        >
          Quotes
        </a>
      </div>
    </nav>
  </div>
</div>
`;
